sdks/fabric/interoperation-node-sdk/test/InteroperableHelper.js

```diff
@@ -300,4 +323,205 @@ describe("InteroperableHelper", () => {
             expect(invokeHandlerResponse).to.equal(true);
         });
     });
+    
+    describe("test interopFlow", () => {
+        const appId = "simplestate";
+        const channel = "mychannel";
+        const appFn = "Write";
+        const appArgs = ["w", "value", "", ""];
+        const argsIndices = [2, 3];
+        const viewAddresses = ["localhost:9080/network1/mychannel:simplestate:Read:Arcturus",
+            "localhost:9080/network1/mychannel:simplestate:Read:Betelguese:1"];
```

Comment:
Same as above.

Reply:
It's already declared constant here right? These are required for local test cases only, is it enough that it's constant within the `describe` test case?

Reply:
I thought the same constant value (or a big chunk of it) was repeated in several locations. If that's the case, it's better to extract it out into a global constant (or set of constants, one for relay address, one for network id, etc.) Otherwise, you can leave it as is.